### DISCUSSION TITLE
jit memory leak

### DISCUSSION THREAD

**Discussion by speakstone** [snakers4/silero-vad#232] [Q&A] [answered]
jit文件，会随着语音的大小增加而增加显存占用，按照推理代码的书写，推测是模型内部存在未释放显存

**Comment by snakers4** [comment, marked as answer]
Please see this:

- https://github.com/snakers4/silero-vad/issues/2#issuecomment-1050968656

**Reply by yangrz7** [reply]
thanks😃
